doc/news/changes/minor/20210819Bangerth

```diff
@@ -0,0 +1,7 @@
+Fixed: It was not possible to default-construct objects of type
+`DoFHandler<dim>::face_iterator` for `dim==1`. As a consequence, one
```

Comment:
```suggestion
`DoFHandler::face_iterator` for `dim==1`. As a consequence, one
```
This might fix the complaint of `doxygen` not finding `face_iterator`.